src/pages/profiles/[publicKey]/nfts.tsx

```diff
@@ -184,81 +184,71 @@ export const NFTCard = ({
           <div
             className={`flex h-full w-full items-end justify-between md:flex-col md:items-center md:justify-between xl:flex-row xl:items-end xl:justify-between`}
           >
+            {!hasDefaultListing && !hasAddedOffer && !Boolean(lastSale) && (
+              <ul className={`mb-0 flex w-full`}>
```

Comment:
Moved the primary button of NFT cards to its own row so it does not conflict with price or other information. It works better in the bigger grid size. Approved by Belle.